app/pages/list/list.js

```diff
@@ -0,0 +1,34 @@
+import {IonicApp, Page, NavController, NavParams} from 'ionic/ionic';
+
+@Page({
+  templateUrl: 'build/pages/list/list.html'
+})
```

Comment:
instead of the @App decorator. It performs a similar function to the @App decorator in that it allows us to define the template to be used, but this time it just sets up the component as a normal page, which will make sure all the required components and services are imported automatically.